wallet/subscription.tact

```diff
@@ -0,0 +1,178 @@
+interface Plugin {
+    internal requested_funds#f06c7567(query_id: Uint64);
+    internal notify_removed#e4737472(query_id: Uint64);
+    internal notify_installed#6e6f7465(query_id: Uint64);
+    internal notify_removed#64737472(query_id: Uint64);
+}
+
+interface Wallet {
+    internal request_funds#706c7567(
+        query_id: Uint64,
+        requested_amount: Coins,
+        requested_extra: ExtraCurrencies,
+        msg.sender: Plugin,
+    );
+    internal remove_plugin#64737472(query_id: Uint64, msg.sender: Plugin);
+}
+
+interface Beneficiary {
+    internal destruct#64737472(msg.sender: Plugin);
+    internal fallback#756e6b77(msg.sender: Plugin);
+    internal subscription#73756273(msg.sender: Plugin);
+}
+
+contract SubscriptionPlugin: Plugin {
+    var wallet: Wallet;
+    var benefeciary: Beneficiary;
+    var amount: Coins;
+    var period: Uint32;
+    var start_time: Uint32;
+    var timeout: Uint32;
+    var last_payment_time: Uint32;
+    var last_request_time: Uint32;
+    var failed_attempts: Uint8;
+    var subscription_id: Uint32;
+
+    internal default() {
+        send(CarryMessageBalance) 
```

Comment:
```suggestion
        send(InboundCoins) 
```
?

Reply:
Nope.
https://github.com/ton-blockchain/wallet-contract/blob/main/func/simple-subscription-plugin.fc#L129-L134
https://github.com/ton-blockchain/wallet-contract/blob/main/func/simple-subscription-plugin.fc#L75-L79

Reply:
Ohh, I understand what you mean. Fixed in [478d31e](https://github.com/tact-lang/tact-bikeshed/pull/7/commits/478d31e269966436cd0128855fca11b5f53eda49)

Reply:
Yeah, there should be `WalletBalance`, not `MessageBalance`. Fixed in the [4b68128](https://github.com/tact-lang/tact-bikeshed/pull/7/commits/4b681286e9e397b293d0ad1e70a1bb9b634950a5)